.github/workflows/codeql.yml

```diff
@@ -0,0 +1,57 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches:
+    - main
+    - 'release/*'
+  pull_request:
+    branches:
+    - main
```

Comment:
why not PRs targeting `release/*` branches?

Reply:
Just took the default one provided by Github.
But I asked myself the same question so I think I will add it